style.css

```diff
@@ -741,27 +748,42 @@ body {
     padding-left: 5%;
   }
 
-  .navlog {
+  .nav-log {
     display: none;
   }
 
-  .nav-desk {
+  .nav-menu {
     display: flex;
+    flex-direction: row;
     list-style: none;
+    position: initial;
+    background-color: #fff;
     text-decoration: none;
```

Comment:
- [ ] Please let's fix this bug, As requested by the previous code reviewer, Your menu icon is still showing on the desktop screen which needs to be hidden. see what happens in the video below :point_down:  when adjusting the width from mobile to desktop view.

[Screencast from 03-07-2023 12:51:21 AM.webm](https://user-images.githubusercontent.com/92303891/223291415-abae0a11-5a62-437f-b4f0-d0d52a871804.webm)

Reply:
@Owusu-Desmond i can’t see the video but in the desktop view the 3 section have to appear but the humbuger menu don’t. Or I’m wrong?

Reply:
I can’t see the video, it’s broken